napalm_eos/eos.py

```diff
@@ -85,6 +85,9 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
 
         self.enablepwd = optional_args.get('enable_password', '')
 
+    def __del__(self):
```

Comment:
I commented on another PR because I had no clue where this was coming from (reading PRs in the wrong order :() but this should be in `napalm-base` to make sure we have a consistent behavior across drivers.